### ISSUE TITLE
Does it work with Single pass for VR?

### ISSUE THREAD

**Issue by keli95566** [ahmaderfani12/PointClouds#1]
Would love to use this in VR. But to make it work with single-pass stereo rendering the shaders would need some changes. 

This would be the reference for what I am talking about:
https://docs.unity3d.com/Manual/SinglePassStereoRendering.html

**Comment by ahmaderfani12** [comment]
Yeah, I got it. Well, it's a bit tricky to implement those instructions in Shader Graph, but I'll put this task in my backlog.
